docs/flashing.md

```diff
@@ -5,22 +5,39 @@ as appropriate (see [Adafruit's
 documentation](https://learn.adafruit.com/welcome-to-circuitpython/installing-circuitpython),
 though it doesn't cover all CircuitPython boards - you may need to glance around
 the CircuitPython source or ask on Discord). We primarily target CircuitPython
-4.0-alpha1 and above, though many features should work on 3.x. You'll only need
+4.0-alpha1 to 4.0-alpha3. You'll only need
```

Comment:
alpha2. Alpha3 introduced the serial changes that break at least split boards, possibly all boards.

Reply:
Whoops